.gitlab/build_toss4.yml

```diff
@@ -57,6 +57,18 @@ toss4-gcc_10_3_1-src:
     ALLOC_DEADLINE: "60"
   extends: .src_build_on_toss4
 
+toss4-clang_16_0_6-src:
+  variables:
+    COMPILER: "clang@16.0.6"
+    HOST_CONFIG: "dane-toss_4_x86_64_ib-${COMPILER}.cmake"
+    EXTRA_CMAKE_OPTIONS: "-DENABLE_BENCHMARKS=ON"
+    EXTRA_BUILD_OPTIONS: "--skip-install"
+    #DO_INTEGRATION_TESTS: "yes"
```

Comment:
```suggestion
```

Reply:
we only do integration tests for one job anyway (when it works), so the comment can be removed

Reply:
fixed